docs/production-deployment/cloud/account-setup/namespaces.mdx

```diff
@@ -152,15 +152,6 @@ Follow these steps to find it:
    Just below that is a row of information including the completed Workflow retention policy, the Namespace region, the Actions-Per-Second limit, and the gRPC endpoint.
 5. Click the copy icon next to the gRPC address to copy it to your clipboard.
 
-<div className="tdiw">
```

Comment:
i noticed the image isn't loading on the docs page so it might be better to just remove this